File: Python/백준/Bronze/5235. Even Sum More Than Odd Sum/Even Sum More Than Odd Sum.py
a = int(input())
for i in range(a):
    aa = list(map(int,input().split()))
    del aa[0]
    b=0;c=0 #홀수합, 짝수합 저장하는 변수
    for j in aa:
        if j%2==1:
            b+=j
        else:
            c+=j
    if(b==c):
        print('TIE')
    elif(b>c):
        print('ODD')
    else:
        print('EVEN')

File: C#/백준/Bronze/5235. Even Sum More Than Odd Sum/Even Sum More Than Odd Sum.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            for (int i=0; i<a; i++){
                int[] aa = Array.ConvertAll(Console.ReadLine().Trim().Split(),int.Parse);
                int esum = 0; //짝수합
                int osum = 0; //홀수합
                for (int j=1; j<aa.Length; j++){
                    if(aa[j]%2==0){
                        esum+=aa[j];
                    }
                    else{
                        osum+=aa[j];
                    }
                }
                if(esum==osum){
                    Console.WriteLine("TIE");
                }
                else if(esum>osum){
                    Console.WriteLine("EVEN");
                }
                else{
                    Console.WriteLine("ODD");
                }
            }
        }
    }
}